manifests/api/service.pp

```diff
@@ -37,5 +37,10 @@
       hasrestart => $hasrestart,
       subscribe  => [ Class['sensu::package'], Class['sensu::api::config'], Class['sensu::redis::config'] ],
     }
+  } elsif $sensu::manage_services and $::sensu::enterprise {
```

Comment:
What about something more like this so we don't declare the sensu-api service twice?

And just to confirm, in SE api should not be running correct?

```puppet
if $sensu::manage_services {

    if $sensu::enterprise {
        $ensure = 'stopped'
        $enable = false
    } elsif $sensu::api {
        $ensure = 'running'
        $enable = true
    } else {
        $ensure = 'stopped'
        $enable = false
    }

    service { 'sensu-api':
      ensure     => $ensure,
      enable     => $enable,
      hasrestart => $hasrestart,
      subscribe  => [ Class['sensu::package'], Class['sensu::api::config'], Class['sensu::redis::config'] ],
    }
  }
```